Source/Flow/Private/FlowSubsystem.cpp

```diff
@@ -265,7 +265,7 @@ TMap<UObject*, UFlowAsset*> UFlowSubsystem::GetRootInstances() const
 	{
 		Result.Emplace(RootInstance.Value.Get(), RootInstance.Key);
 	}
-	return Result;
```

Comment:
_Explicitly calling MoveTemp() to invoke move semantics.  (yes, compilers will often detect this optimization possibility, but I like being explicit, particularly when it involves a deep copy of an array)._